test/Microsoft.Framework.Internal.Test/PropertyActivatorTest.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.Framework.Internal
+{
+    public class PropertyActivatorTest
+    {
+        [Fact]
+        public void Activate_InvokesValueAccessorWithExpectedValue()
+        {
+            // Arrange
+            var instance = new TestClass();
+            var typeInfo = instance.GetType().GetTypeInfo();
+            var property = typeInfo.GetDeclaredProperty("IntProperty");
+            var invokedWith = -1;
+            var activator = new PropertyActivator<int>(
+                property,
+                valueAccessor: (val) =>
+                {
+                    invokedWith = val;
+                    return val;
+                });
+
+            // Act
+            activator.Activate(instance, 123);
+
+            // Assert
+            Assert.Equal(123, invokedWith);
+        }
+
+        [Fact]
+        public void Activate_SetsPropertyValueBasedOnContext()
```

Comment:
what context is used in this test?  main difference from `Activate_InvokesValueAccessorWithExpectedValue()` is check of activated value.  suggest either adding an assertion to the above test or rename this to `Activate_SetsPropertyValue()`.